test/test_video_reader.py

```diff
@@ -0,0 +1,1042 @@
+import collections
+from common_utils import get_tmp_dir
+from fractions import Fraction
+import logging
```

Comment:
I haven't seen logging used in torchvision in general. What is the best prActice for this @fmassa?

Reply:
We don't currently use logging in torchvision, only emit some deprecation warnings at some places.

I'm not yet sure we want to add logging as of now, it might deserve a larger discussion

Reply:
Ok. Those logging are mostly used for my dev. I will remove them now.